micrometer/src/test/java/feign/micrometer/FeignHeaderInstrumentationTest.java

```diff
@@ -82,9 +88,49 @@ void getTemplatedPathForUriForAsync(WireMockRuntimeInfo wmRuntimeInfo)
     testClient.templated("1", "2").get();
 
     verify(getRequestedFor(urlEqualTo("/customers/1/carts/2")).withHeader("foo", equalTo("bar")));
-    Timer timer = meterRegistry.get("http.client.requests").timer();
+    Timer timer = meterRegistry.get(METER_NAME).timer();
     assertThat(timer.count()).isEqualTo(1);
     assertThat(timer.totalTime(TimeUnit.NANOSECONDS)).isPositive();
+
+    assertTags();
+  }
+
+  private void assertTags() {
+    Id requestsId = meterRegistry.get(METER_NAME).meter().getId();
+
+    assertMetricIdIncludesMethod(requestsId);
+    assertMetricIdIncludesURI(requestsId);
+    assertMetricIdIncludesStatus(requestsId);
+    assertsMetricIdIncludesClient(requestsId);
+  }
```

Comment:
This might not be the right place for these tests, however since they're only applicable to the micrometer observation meters adding to [AbstractMetricsTestBase.java](https://github.com/OpenFeign/feign/blob/master/micrometer/src/test/java/feign/micrometer/AbstractMetricsTestBase.java) would probably require some changes to the dropwizard 4/5 and MicrometerCapability tests